cobigen/cobigen-core/src/main/java/com/devonfw/cobigen/impl/util/TimestampUtil.java

```diff
@@ -0,0 +1,139 @@
+package com.devonfw.cobigen.impl.util;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.sql.Timestamp;
+import java.util.Date;
+import java.util.Properties;
+
+import org.apache.commons.lang3.time.DateUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.devonfw.cobigen.api.constants.ConfigurationConstants;
+import com.devonfw.cobigen.api.util.CobiGenPaths;
+
+/**
+ * This Class is used to set the timestamp property
+ *
+ */
+public class TimestampUtil {
```

Comment:
odd naming as this is only writing CONFIG_PROPERTY_TIME_STAMP (which is also an odd name) - what is the timestamp about? -> name it what it is (business - not technical)